enhancements/ingress/host-network-configurable-ports.md

```diff
@@ -258,9 +287,11 @@ N/A.
 
 ## Drawbacks
 
-N/A.
+Additional operational complexity, which will increase the needed test matrix, frustration for users, and volume of customer cases.
 
 ## Alternatives
 
-N/A.
+* Advise users to the NodePortService endpoint publishing strategy type. 
+* Augment the NodePortService strategy to allow the user to specify requested ports within the nodeport range. 
```

Comment:
Usually this section will include some explanation of why the alternatives were rejected.  Do we have that information?  Maybe it's as simple as that these alternatives have been rejected by a subset of users who require HostNetwork for unspecified reasons.  If we do know a reason why these alternatives were rejected, this would be the place to record it.